Tensile/Configs/rocblas_sgemm_nt_inc1_asm_full.yaml

```diff
@@ -322,12 +482,315 @@ BenchmarkProblems:
           - Exact: [ 67, 2048, 1, 512, 67, 67, 67, 2048 ]
           - Exact: [ 74, 2048, 1, 512, 74, 74, 74, 2048 ]
           - Exact: [ 768, 2048, 1, 256, 768, 768, 768, 2048 ]
+          - Exact: [ 33, 3280, 1600, 33, 33, 33, 33, 32 ]
+          - Exact: [ 33, 3280, 200, 33, 33, 33, 33, 32 ]
+          - Exact: [ 3, 64, 512, 3, 3, 3, 3, 64 ]
+          - Exact: [ 5, 64, 512, 5, 5, 5, 5, 64 ]
+          - Exact: [ 9, 64, 512, 9, 9, 9, 9, 64 ]
+
+    - # BenchmarkProblemSizeGroup - Standard
+      InitialSolutionParameters:
+      BenchmarkCommonParameters:
+        - KernelLanguage: ["Assembly"]
+        - EdgeType: ["ShiftPtr"]
+        #- LoopTail: [True]
+        - PrefetchLocalRead: [True]
+      ForkParameters:
+        - MatrixInstruction:
+          - [16, 16, 4, 1]
```

Comment:
From this point onward, I had to delete all the sizes due to an assertion error. I got numElementsPerBatch=0 for 31 kernels for the 16x16x4x1 matrix instruction. It's difficult to tell which combinations I have to remove, so I removed all of them. I didn't have this issue for the nn file, and the tn one is still running for me